Jenkinsfile

```diff
@@ -101,6 +102,23 @@ node('ubuntu-zion') {
         gitHub.statusUpdate commitId, 'success', 'test', 'Tests succeeded'
       }
     }
+    stage('Evaluate') {
+      
+      //Create tar of our image
+      OsTools.runSafe(this, "docker save ${imageName} -o ${env.WORKSPACE}/${tarName}")
+      
+      //decide which stage we are creating
+      def theStage = 'build'
+      if(branch == 'master') {
+        theStage = 'release'
+      }
+
+      //run the evaluation
+      nexusPolicyEvaluation iqStage: "${theStage}", iqApplication: "${archiveName}",
```

Comment:
It shouldn't be necessary to use Groovy string interpolation here.  You can access the variables directly.

```groovy
nexusPolicyEvaluation iqStage: theStage, iqApplication: archiveName,
```

Reply:
Fixed in b6aaba7